backend/src/auth/mod.rs

```diff
@@ -326,6 +367,24 @@ impl HasRoles for User {
     }
 }
 
+impl HasRoles for AuthContext {
+    fn roles(&self) -> &[String] {
+        static TRUSTED_ROLES: Lazy<[String; 1]> = Lazy::new(|| [ROLE_ANONYMOUS.into()]);
```

Comment:
Trusted external tools only have the `ROLE_ANONYMOUS`? My intuition would be that they are rather powerful? Can you explain this again?